code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -107,6 +107,70 @@
 		message = pick("NEEIIGGGHHHH!", "NEEEIIIIGHH!", "NEIIIGGHH!", "HAAWWWWW!", "HAAAWWW!")
 	return message
 
+/obj/item/clothing/mask/rat

```

Comment:
these all have the same `flags_inv` and `flags_cover`.

Why not make them all a child of a single type to avoid copy-paste? If there's an exception, eventually, you can just override it.